pkg/scheduler/apis/config/types.go

```diff
@@ -94,6 +94,11 @@ type KubeSchedulerConfiguration struct {
 	// the default value (10s) will be used.
 	PodMaxBackoffSeconds int64
 
+	// UnschedulableQTimeIntervalSeconds is a pod's maximum duration in unschedulableQ.
+	// If specified, it must be greater than 0. If this value is null, the default value (60s)
+	// will be used.
+	UnschedulableQTimeIntervalSeconds int64
```

Comment:
If we are to use the same naming pattern as above, then lets perhaps name it: `PodMaxUnschedulableSeconds`

Reply:
it seems it's a duplicate of https://github.com/kubernetes/kubernetes/pull/101698. I'm closing this one.